docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -166,135 +160,27 @@ If multiple custom converters for a type are registered in the `Converters` coll
-
-The following code shows a base class, two derived classes, and a custom converter for them. The converter uses a discriminator property to do polymorphic deserialization. The type discriminator isn't in the class definitions but is created during serialization and is read during deserialization.
-
-[!code-csharp[](~/samples/snippets/core/system-text-json/csharp/Person.cs?name=SnippetPerson)]
-
-[!code-csharp[](~/samples/snippets/core/system-text-json/csharp/PersonConverterWithTypeDiscriminator.cs)]
-
-The following code registers the converter:
-
-[!code-csharp[](~/samples/snippets/core/system-text-json/csharp/ConvertPolymorphic.cs?name=SnippetRegister)]
-
-The converter can deserialize JSON that was created by using the same converter to serialize, for example:
-
-```json
-[
-  {
-    "TypeDiscriminator": 1,
-    "CreditLimit": 10000,
-    "Name": "John"
-  },
-  {
-    "TypeDiscriminator": 2,
-    "OfficeNumber": "555-1234",
-    "Name": "Nancy"
-  }
-]
-```
-
 ## Other custom converter samples
 
-The [unit tests folder](https://github.com/dotnet/corefx/blob/master/src/System.Text.Json/tests/Serialization/) in the `System.Text.Json.Serialization` source code includes other custom converter samples, such as:
+The [Migrate from Newtonsoft.Json to System.Text.Json](system-text-json-migrate-from-newtonsoft-how-to.md) article contains samples of custom converters.
+
+The [unit tests folder](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Text.Json/tests/Serialization/) in the `System.Text.Json.Serialization` source code includes other custom converter samples, such as:
 
 * `Int32` converter that converts null to 0 on deserialize
 * `Int32` converter that allows both string and number values on deserialize
 * `Enum` converter
 * `List<T>` converter that accepts external data
 * `Long[]` converter that works with a comma-delimited list of numbers 
 
+If you need to make a converter that modifies the behavior of an existing built-in converter, you can get [the source code of the existing converter](https://github.com/dotnet/runtime/tree/master/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters) to serve as a starting point for customization.
+
 ## Additional resources
 
+* [GitHub issue related to custom converters](https://github.com/dotnet/corefx/issues/36639)
+* [Source code for built-in converters](https://github.com/dotnet/runtime/tree/master/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/)
```

Comment:
@steveharter can correct me if I'm wrong, but I think people will want to look at the built-in converters as samples since they can't make derived classes based on them.

Reply:
Fair point. I am fine with keeping this link. I would update the link to be a fixed commit hash though:
https://github.com/dotnet/runtime/tree/81bf79fd9aa75305e55abe2f7e9ef3f60624a3a1/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters